clang/lib/CodeGen/Targets/ARM.cpp

```diff
@@ -141,7 +141,7 @@ class ARMTargetCodeGenInfo : public TargetCodeGenInfo {
       ParsedTargetAttr Attr =
           CGM.getTarget().parseTargetAttr(TA->getFeaturesStr());
       if (!Attr.BranchProtection.empty()) {
-        TargetInfo::BranchProtectionInfo BPI;
+        TargetInfo::BranchProtectionInfo BPI{};
```

Comment:
These are unnecessary now, right? Making them explicit is fine though.

Reply:
Right, now unnecessary, maybe more stylish.